packages/core/src/startup/StartupSystem.ts

```diff
@@ -0,0 +1,48 @@
+import { sayHello } from '@pixi/utils';
+import type { BackgroundOptions } from '../background/BackgroundSystem';
+import type { ViewOptions } from '../view/ViewSystem';
+import type { IRendererPlugins } from '../plugin/PluginSystem';
+import { IRenderer } from '../IRenderer';
+import { ISystem } from '../system/ISystem';
+
+// TODO this can be infered by good use of generics in the future..
+export interface StartupOptions extends Record<string, unknown> {
+    _plugin: IRendererPlugins,
+    background: BackgroundOptions,
+    _view: ViewOptions,
+}
```

Comment:
I see the todo to make this a generic which is a good idea

However, this is missing an optional `ContextOptions` for use in `Renderer`

Reply:
cool, added!